### DISCUSSION TITLE
Our other github account was hacked

### DISCUSSION THREAD

**Discussion by BimoSora2** [community/community#39283] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Our other github account was hacked.  We don't know why, there was an incoming notification in the email but suddenly my account was hit by a pornography violation.  We don't know what happened.  2 months account not opened and suddenly hackers enter, account banned.

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Comment by ng-druid** [comment]
Did the account have mfa enabled that was hacked?

**Reply by elizabethcabugnason385-dotcom** [reply]
Yed

**Comment by khan-tutul** [comment]
Unsubscribe

**Comment by elizabethcabugnason385-dotcom** [comment]
I need hacks pls

**Comment by elizabethcabugnason385-dotcom** [comment]
roblox hacks pls
